app/views/games/_team_list.html.haml

```diff
@@ -30,4 +30,7 @@
     - if @view_all_teams_link
       %tr
         %td{align: 'center', colspan: '5', style: 'text-align: center;'}
-          = link_to 'View all Teams', game_teams_path
+          - if !display_all
+            = link_to 'Show More Teams', game_summary_path(:display_all => "true")
+          - else
```

Comment:
There is a setting in your editor to automatically add new lines to the end of a file, generally the best practice is to have that enabled: https://thoughtbot.com/blog/no-newline-at-end-of-file